CONTRIBUTING.md

```diff
@@ -0,0 +1,223 @@
+# Nodejs Loaders Contributing Guide
+
+Thank you for considering contributing to Nodejs Loaders! We welcome all contributions, whether they are bug reports, feature requests, pull requests, or just questions.
+
+## Getting started
+
+The steps below will give you a general idea of how to prepare your local environment for nodejs loaders development.
+
+1. Click the fork button in the top right to clone the [Node.js Website Repository](https://github.com/nodejs-loaders/nodejs-loaders/fork)
+
+2. Clone your fork using SSH, GitHub CLI, or HTTPS.
+```bash
+git clone git@github.com:<YOUR_GITHUB_USERNAME>/nodejs-loaders.git # SSH
+git clone https://github.com/<YOUR_GITHUB_USERNAME>/nodejs-loaders.git # HTTPS
+gh repo clone <YOUR_GITHUB_USERNAME>/nodejs-loaders.org # GitHub CLI
+```
+
+3. Change into the directory.
+```bash
+cd nodejs-loaders
+```
+
+4. Create a remote to keep your fork and local clone up-to-date.
+```bash
+git remote add upstream git@github.com:nodejs-loaders/nodejs-loaders.git # SSH
+git remote add upstream https://github.com/nodejs-loaders/nodejs-loaders.git # HTTPS
+gh repo sync nodejs-loaders/nodejs-loaders # GitHub CLI
```

Comment:
I think the ssh one is better? If we're going to include them, I would include just that one.

Reply:
honstly idk I'm using GitHub desktop

Reply:
I do to (for like 70% of stuff), but I always set up my repos via CLI.